public/v4/apps/authentik.yml

```diff
@@ -0,0 +1,179 @@
+captainVersion: 4
+services:
+    $$cap_appname-postgresql:
+        image: docker.io/library/postgres:$$cap_postgres_version-alpine
+        restart: unless-stopped
+        healthcheck:
+            test:
+                [
+                    'CMD-SHELL',
+                    'pg_isready -d $$cap_postgres_db -U $$cap_postgres_user',
+                ]
+            start_period: 20s
+            interval: 30s
+            retries: 5
+            timeout: 5s
+        volumes:
+            - $$cap_appname-postgresql-data:/var/lib/postgresql/data
+        environment:
+            POSTGRES_PASSWORD: $$cap_postgres_password
+            POSTGRES_USER: $$cap_postgres_user
+            POSTGRES_DB: $$cap_postgres_db
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+
+    $$cap_appname-redis:
+        image: docker.io/library/redis:alpine
```

Comment:
This is technically a "latest" tag. It's a rolling update and might break in the future. Can we fix to a specific version?